documentation/tutorial.md

```diff
@@ -109,6 +109,14 @@ some functions:
 3. parse_item(): function which will be called to handle response
    downloaded by Crawly. It must return the `Crawly.ParsedItem` structure.
 
+As it is, the init function returns an empty array for `start_urls` which we will
+fix now. If you already know what urls, you want to start crawling, you can place them in
+the array, else you can pick the urls you want to start with from a particular page
+on your website of interest.
+
+Using the shell, you can try selecting elements using Floki with the response. That gives you a faster way to test out your selectors
```

Comment:
Mark this as a sidenote `using `>` blockquotes.

> Note: You can use the `iex` shell to test your selectors on the response HTML using Floki. Alternatively, use your browser's developer tools to test your selector.

Reply:
Done